src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+.logo_link {
+  height: 40px;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__menu {
+  display: flex;
+  list-style: none;
+  height: 60px;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  height: 100%;
+}
```

Comment:
CSS: Avoid fixing container size. Let the content size dictate it. This is to avoid overflow or accidental scroll bars. Here, you've fixed the height of `.nav__item`, which might lead to overflow if the content inside it exceeds the height.